docs/nextflow-create-docker.md

```diff
@@ -78,7 +79,7 @@ Once you are signed in to Docker, you can run the command they suggest after pul
 
 You can run the next suggested command (shown in red box above, `docker scout cves...`) to see the full list of vulnerabilities.
 
-Images will be able to pass Gen3 security scanning if there are no Critical or High vulnerabilities, and **[add something about CVSS?]**
+Images should be able to pass Gen3 security scanning if there are no Critical vulnerabilities.
```

Comment:
THank you! I was really trying to figure this out and couldn't find the answer anywhere.

Reply:
i accepted this change in a new pr